config.md

```diff
@@ -57,13 +57,13 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
   * Windows: one mount destination MUST NOT be nested within another mount (e.g., c:\\foo and c:\\foo\\bar).
   * Solaris: corresponds to "dir" of the fs resource in [zonecfg(1M)][zonecfg.1m].
 * **`type`** (string, OPTIONAL) The filesystem type of the filesystem to be mounted.
-  * Linux: valid *filesystemtype* supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
+  * Linux: valid *filesystem type* supported by the kernel as listed in */proc/filesystems* (e.g., "minix", "ext2", "ext3", "jfs", "xfs", "reiserfs", "msdos", "proc", "nfs", "iso9660").
```

Comment:
> waiting any comments in #813.

Briefly covered in the [meeting][1] with @mrunalp and @crosbymichael, this is not a statement about config validation.  The spec has no opinions on what strings are valid values here.  Once some RFC 2119 language lands (e.g. see [here][2] in the rejected #771, which I hope to revive after opencontainers/runc#1442 lands and I can catch runC up with current kernels), it will be clear that all the runtime has to do is hand this value off to the kernel.  Whether the mount work or not is between the config author and the kernel.  So we want to drop `valid` here; this line is just a hint to config authors about where to figure out what the kernel will accept, and has nothing to do with config validation or runtime compliance testing.

[1]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2017/opencontainers.2017-05-12-21.01.log.html#l-40
[2]: https://github.com/opencontainers/runtime-spec/pull/771/files#diff-c9c91c29b41257aea3a3403cc606ad99R64